src/bun.js/api/brotli.zig

```diff
@@ -0,0 +1,569 @@
+const bun = @import("root").bun;
+const JSC = bun.JSC;
+const std = @import("std");
+const brotli = bun.brotli;
+
+const Queue = std.fifo.LinearFifo(JSC.Node.BlobOrStringOrBuffer, .Dynamic);
+
+fn ConcurrentByteProcessor(comptime Processor: type) type {
+    _ = Processor; // autofix
+    return struct {};
+}
+
+pub const BrotliEncoder = struct {
```

Comment:
I think one thing we're going to need to do here that will be tricky is reporting the memory allocation size to GC